sgl-kernel/csrc/attention/merge_attn_states.cu

```diff
@@ -0,0 +1,191 @@
+// Adapted from https://github.com/vllm-project/vllm/pull/16173
```

Comment:
`merge_attn_states` in both engines are written by you. Do we still need to keep this `Adapted` comment

Reply:
done